src/renderer/modals/certificate/certificate.jsx

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import {MODAL_CANCEL, MODAL_RESULT, RETRIEVE_MODAL_INFO} from 'common/communication.js';
+
+import SelectCertificateModal from './certificateModal.jsx';
+
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Yes, if you override the default bootstrap `body` `background-color` to `transparent` it will remove the white background, which is what i've done here.

Reply:
good to know, should we fix it in the new server modal? (in a separate PR)

Reply:
Yeah we could do that.